src/interfaces/Members.ts

```diff
@@ -21,6 +55,56 @@ export const CATALYSTS = [
   'peer.dclnodes.io',
 ]
 
+export interface DCLProfile {
+  timestamp: number
+  avatars: Avatar[]
+}
+
+interface Avatar {
+  userId?: string
+  email?: string
+  name?: string
+  hasClaimedName: boolean
+  description: string
+  ethAddress?: string
+  version?: number
+  avatar: AvatarConfig
+  tutorialStep?: number
+  interests?: any[]
+  unclaimedName?: string
+  inventory?: string[]
+  snapshots?: Snapshots
+  hasConnectedWeb3?: boolean
+  muted?: string[]
+  blocked?: string[]
+}
+
+interface AvatarConfig {
+  bodyShape: string
+  snapshots: Snapshots
+  eyes: Eyes
+  hair: Eyes
+  skin: Eyes
+  wearables: string[]
+  version?: number
+}
+
+interface Eyes {
+  color: Color
+}
+
+interface Color {
+  r: number
+  g: number
+  b: number
+  a: number
+}
+
+interface Snapshots {
+  body: string
+  face256: string
+}
+
 export const STRATEGIES = [
```

Comment:
I think wearable strategy is missing here (also inside delegation strategy)

Reply:
Yes I know, but that requires a separate PR since all the scripts that use VP must be modified :/